### DISCUSSION TITLE
< quality LinkedIn post generator Need inspiration >

### DISCUSSION THREAD

**Discussion by ShahFH** [community/community#131694] [Programming Help]
### Select Topic Area

Question

### Body

I am working on LinkedIn post & headline Generator. I would love any insight you might have or would you be interested in contributing.

Thank you

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @ShahFH ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
